init/bootrr-auto.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if $(grep -q bootrr-auto /proc/cmdline); then
+	for TEST in $(tr "\0" "\n" < /proc/device-tree/compatible); do
+		if [ -x "/usr/bin/${TEST}" ]; then
```

Comment:
/usr/bin here make the assumption that prefix was set to /usr. we need to adjust based on $prefix.

Reply:
@ndechesne @andersson I just added a replacement take into account DESTDIR and prefix in the file.